### PR TITLE
Add MoFo pages

### PR DESCRIPTION
**Pull request by TheoChevalier**
No rush here, won’t merge until Saturday

### CODE REVIEW

File: en-US/foundation/advocacy.lang [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+;Mozilla’s advocacy approach incorporates both scale to depth and depth to scale to properly address the diversity of issues that we work on.
+Mozilla’s advocacy approach incorporates both scale to depth and depth to scale to properly address the diversity of issues that we work on.
+
+
+;This approach enables Mozilla to simultaneously build our capacity to mobilize millions of people quickly and grow our organizing power in local communities, which will make it possible to win on our issues in the long-term.
+This approach enables Mozilla to simultaneously build our capacity to mobilize millions of people quickly and grow our organizing power in local communities, which will make it possible to win on our issues in the long-term.
+
+
+;We also push ourselves to be proactive—setting the topic and tone of the conversation, and promoting solutions—as well as reactive to current events or undesirable policies when necessary.
+We also push ourselves to be proactive—setting the topic and tone of the conversation, and promoting solutions—as well as reactive to current events or undesirable policies when necessary.
```
Comment (flodolo) [inline en-US/foundation/advocacy.lang]: These - without spaces before and after don't look right (they have them in the leadership file).

Comment (TheoChevalier) [inline en-US/foundation/advocacy.lang]: Yeah, noticed, but at this point I consider that as a en-US issue that would be fixed anyway in the translations. But if you think it’s worth fixing, I can spin a quick Bedrock patch

Comment (flodolo) [inline en-US/foundation/advocacy.lang]: I'm tempted to say that we should fix it in bedrock, and pre-fix the string here.

Comment (TheoChevalier) [inline en-US/foundation/advocacy.lang]: k, pre-fixed here

**Pull request merged**